android-ktx/src/main/java/com/mirego/trikot/viewModels/adapter/NoDiffCallback.kt

```diff
@@ -0,0 +1,10 @@
+package com.mirego.trikot.viewmodels.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+
+class NoDiffCallback<T> : DiffUtil.ItemCallback<T>() {
+
+    override fun areItemsTheSame(oldItem: T, newItem: T) = false
```

Comment:
NoDiffCallback devrait pas retourner true tout le temps, sans différence tout est pareille non?  ou bien ce serait plus AlwaysDiffCallback (ca sonne pas si bien....) ?

Reply:
I'll keep the original naming as I wasn't able to find a better name NoDiffCallback has been a term we used internally in a few projects already